setup.cfg

```diff
@@ -51,14 +51,13 @@ python_requires = >= 3.7
 packages =
 
 [flake8]
+# https://flake8.readthedocs.org/en/latest/
+exclude=\
+    venv*/*,\
+    local/*,\
+    build/*,\
+    .eggs/*,\
+    .tox/*
 max-complexity = 25
 max-line-length = 119
-
-[isort]
-profile = black
-include_trailing_comma = false
-split_on_trailing_comma = true
-combine_as_imports = true
-ensure_newline_before_comments = false
```

Comment:
Why were these settings removed? It's awkward to see what effect this has had because they're squashed into the same commit as Black, but you can see some examples in src/winforms/src/toga_winforms/libs, and I don't think it's an improvement.

Note that `split_on_trailing_comma` requires the pre-release isort version which was previously in .pre-commit-config.yaml.

Reply:
I originally removed them as part of the "trying to get isort, black and flake8 to play nice with each other" struggle. 

After I'd made all the other simplifications from your review (mostly consolidating into a single root-level config, and disabling the isort/flake exclusions), I re-added these in one by one to see the effect.

`include_trailing_comma = false` and `ensure_newline_before_comments = false` are in direct opposition to black style; black *always* includes the trailing comma, and puts a blank line before top-level comments. 

However, I agree `combine_as_imports = true` and `split_on_trailing_comma=true` are improvements, though; black seems has no opinion on the former, and the latter seems to be more of a "make isort behave like black" option (it's moderately surprising to me it's not included in the "black" default profile).

So - I've restored those last two, and I've audited the resulting `libs` and `factory` modules in each of the backends (as they're the modules that are most impacted by import formatting changes). I'm happy enough with the formatting that is there now; let me know if you disagree.